index.html

```diff
@@ -16,9 +19,48 @@
   <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-  <header>
+  <header class="header-container">
+    <div class="container">
+      <h2 class="header-text">Code Journal</h2>
+    </div>
   </header>
   <main>
+    <div class="container">
+      <div>
+        <h1 class="main-header">New Entry</h1>
+      </div>
+      <div data-view="entry-form">
+        <form class="entry-form-container" action="index.php" method="POST">
+          
+          <div class="row top-form">
+            <div class="entry-form-div column-half">  
+              <img src="images/placeholder-image-square.jpg" alt="Placeholder Image" class="entry-image" id="entry-image">
+            </div>
+            
+            <div class="top-inputs column-half">
+              <div class="entry-form-div">
+                <div><label for="entry-title" class="label-text">Title</label></div>
+                <div><input required autofocus type="text" name="entry-title" id="entry-title" placeholder="Title" class="input-width"></div>
+              </div>
+
+              <div class="entry-form-div">
+                <div><label for="photo-url" class="label-text">Photo URL</label></div>
+                <div><input required autofocus type="url" name="photo-url" id="photo-url" placeholder="https://" class="input-width"></textarea></div>
+              </div>
+            </div>
+          </div>
+          
+          <div class="entry-form-div">
+            <div><label for="entry-notes" class="label-text">Notes</label></div>
+            <div><textarea required autofocus name="entry-notes" id="entry-notes" placeholder="Notes" class="notes-area notes-area-height"></textarea></div>
+          </div>
+
+          <div class="entry-form-div">
+            <button type="submit" name="submit-entry" id="save-button" class="submit-entry-button">SAVE</button>
+          </div>
+        </form>
+      </div>
+    </div>
```

Comment:
please update the indentation so that the elements line up and the white space between elements is removed.